frontend/src/components/AcmMasonry.tsx

```diff
@@ -2,7 +2,7 @@
 import { Grid, GridItem, gridSpans, Stack } from '@patternfly/react-core'
 import useResizeObserver from '@react-hook/resize-observer'
 import { Children, Dispatch, ReactNode, SetStateAction, useLayoutEffect, useMemo, useRef, useState } from 'react'
-import { AcmLoadingPage } from '../ui-components'
+import { LoadingPage } from './LoadingPage'
```

Comment:
Why did you change this? Looks like they are pretty much identical, but AcmLoadingPage has translation.